src/Adapters/Log4NetAppender.Net45/ApplicationInsightsAppender.cs

```diff
@@ -142,7 +142,7 @@ private void SendTrace(LoggingEvent loggingEvent)
         private void BuildCustomProperties(LoggingEvent loggingEvent, ITelemetry trace)
         {
             trace.Timestamp = loggingEvent.TimeStamp;
-            trace.Context.User.Id = loggingEvent.UserName;
+            trace.Context.User.AuthenticatedUserId = loggingEvent.UserName;
```

Comment:
Possible future work: According to GDPR we might want to limit the collection of user account information in auto-collection scenarios and rather resort to anonymous identification. If message contains actual user ID - then it's fine and it's customer's choice to trace it, if we infer actual user ID from the data and store it - we will need to provide ability for customer to know about this data being collected and ability to clear it on backend.

Reply:
Yep. Keeping it as is for now.